index.js

```diff
@@ -1,5 +1,6 @@
 var stream = require('stream')
   , mime = require('mime')
+  , mimeTypes = require('mime-types')
```

Comment:
If we're moving to this new module I'd like to see us remove the old one entirely.